<launch>
  
  <node respawn="true" pkg="joy" type="joy_node" name="joy"/>

  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular_z" value="0" type="int" />
  <param name="axis_angular_x" value="2" type="int" />
  <param name="axis_angular_y" value="3" type="int" />
  <param name="scale_linear" value="10" type="double" />
  <param name="scale_angular_x" value="-0.707" type="double" />
  <param name="scale_angular_y" value="-0.707" type="double" />
  <param name="scale_angular_z" value="-1.57" type="double" />
  <node pkg="joy_ctrl" type="joy_teleop" name="teleop"/>

</launch>

